RPIPs/RPIP-46.md

```diff
@@ -29,76 +29,83 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
+Prior to the release of Saturn 1, a ranked-choice vote MUST be held to select a mechanism for revenue share to RPL
+1. The choices SHALL include retaining the state from Saturn 1
+2. The choices MAY include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md), and novel options created for this vote
+   1. The choices SHOULD specify a new share (eg, `buy_and_burn_share`) with an initial value
+   2. The choices SHOULD specify a reduction of other shares (eg, `voter_share`) to balance the new share 
+3. The selected mechanism MUST be implemented in Saturn 2
+4. If the state from Saturn 1 is retained, the entirety of the ["Implementing the revenue share vote"](#implementing-the-revenue-share-vote) section below SHALL be deleted
+
+## Specification taking effect with Saturn 2
+### RPL rewards and inflation
 1. Inflation settings SHALL be modified to retain inflation to the DAOs and eliminate inflation to NOs
    1. `rpl.inflation.interval.rate` SHALL be set to `1000040763630249500` (1.5% per year)
    2. Node Operators (`rocketClaimNode`) allocation SHALL be set to 0%
    3. pDAO (`rocketClaimDAO`) allocation SHALL be set to 95%
    4. oDAO (`rocketClaimTrustedNode`) allocation SHALL be set to 5%
-
-### Surplus revenue share vote
-Prior to the release of Saturn 1, a ranked-choice vote MUST be held to select a mechanism for surplus revenue share
-1. The choices MAY include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md), and using higher `voter_share`
-2. If higher `voter_share` is selected:
-   1. The entirety of the ["Specification taking effect with Saturn 2"](#specification-taking-effect-with-saturn-2) section below SHALL be deleted 
-   2. `voter_share` MAY be updated by an address in the `allowlisted_controllers` array
-   3. `node_operator_commission_share_council_adder` shall be subtracted from `voter_share` instead of `surplus_share`
-   4. `voter_share` SHALL be a pDAO setting (and thus the specification item about it not being controllable by pDAO vote MUST also be removed)
-3. The selected mechanism MAY be implemented in Saturn 1
-   1. If not, the revenue SHALL be held in reserve until the mechanism is implemented. In such a case, once the mechanism is implemented, the revenue SHALL be distributed in the same amount of time it took to build up, or faster.
-4. The selected mechanism MUST be implemented in Saturn 2
-5. When implemented, `surplus_share` SHOULD be renamed to something more specific
-
-## Specification taking effect with Saturn 2
-### Updating `voter_share`:
-The `voter_share` SHALL be adjustable, to target a share of voting eligible RPL (including 
-megapool staked RPL and legacy staked RPL) relative to the total supply of RPL minus RPL 
-owned by the protocol in its treasury or as a result of `surplus_share` buy backs. The voting
-eligible RPL target is a range between `vote_eligible_target_min` and `vote_eligible_target_max`
-1. Updating `voter_share`:
+2. There SHALL be no RPL rewards 
```

Comment:
```suggestion
2. There SHALL be no RPL issuance to NOs 
```

Reply:
(voter_share, or other revenue sharing to RPL could generically be considered as "RPL rewards")